static/app/components/profiling/ProfilingOnboarding/profilingOnboardingModal.tsx

```diff
@@ -368,7 +368,11 @@ function AndroidInstallSteps({
 }: InstallStepsProps) {
   const hasSdkUpdates = sdkUpdates.type === 'resolved' && sdkUpdates.data !== null;
   const requiresSdkUpdates =
-    hasSdkUpdates && sdkUpdates.data?.sdkVersion
+    hasSdkUpdates &&
+    sdkUpdates.data?.sdkVersion &&
+    ['sentry.java.android', 'sentry.java.android.timber'].includes(
```

Comment:
Lgtm, but can we just add a comment here about why we are doing that so it looks less magical :D